xml/System.IO/BufferedStream.xml

```diff
@@ -1097,10 +1142,22 @@ bufStream->Close();
         <Parameter Name="destination" Type="System.Span&lt;System.Byte&gt;" Index="0" FrameworkAlternate="netcore-3.0" />
       </Parameters>
       <Docs>
-        <param name="destination">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="destination">A region of memory. When this method returns, the contents of this region are replaced by the bytes read from the current source.</param>
+        <summary>Reads a sequence of bytes from the current buffered stream and advances the position within the buffered stream by the number of bytes read.</summary>
```

Comment:
The other overload is phrased as "Copies bytes..." Perhaps make this parallel -- "Copies a sequence of bytes from the current buffered stream to a byte span and advances..."?

Reply:
```suggestion
        <summary>Copies bytes from the current buffered stream to a byte span and advances the position within the buffered stream by the number of bytes read.</summary>
```